docs/start/basics.ipynb

```diff
@@ -611,7 +611,7 @@
     "id": "B8ciZZSSf2jb"
    },
    "source": [
-    "Another example is the 3-qubit Toffoli gate, which is equivalent to a controlled-controlled-X gate.  Many devices do not support a three qubit gate, so it is important "
+    "Another example is the 3-qubit Toffoli gate, which is equivalent to a controlled-controlled-X gate.  Many devices do not support a three qubit gate, so it is important to decompose this into one and two qubit gates."
```

Comment:
Suggestion:

Many devices do not support three qubit gates, so it is important to decompose them into one and two qubit gates.

(Removes "a", changes "gate" to "gates" and "this" to "them".)